engine/src/cmd/ai/hard_coded_scripts.cpp

```diff
@@ -889,7 +889,8 @@ void Takeoff( Order *aisc, Unit *un )
     ord = ( new Orders::FaceTargetITTS( 0, 3 ) );
     AddOrd( aisc, un, ord );
     un->SelectAllWeapon( false );
-    un->Fire( (weapon_info::LIGHT || weapon_info::MEDIUM), false );
```

Comment:
I think this is supposed to be the bitwise OR operator `|`, rather than the logical OR operator `||`. I.e.,

```C++
un->Fire( (weapon_info::LIGHT | weapon_info::MEDIUM), false );
```

Reply:
You might be right, of course. My reasoning here was that this un->Fire thing would be done if either of those weapon_info's evaluated to true, so why not run them after another? I will update to your suggestion, and see if that makes the compile error go away as well. 
I'm pretty sure your suggestion is closer to the original intent.